src/inmanta/protocol/methods_v2.py

```diff
@@ -1063,24 +1076,26 @@ def get_parameters(
 ) -> List[model.Parameter]:
     """
     List the parameters in an environment
+
     :param tid: The id of the environment
     :param limit: Limit the number of parameters that are returned
     :param first_id: The parameter id to use as a continuation token for paging, in combination with the 'start' value,
-            because the order by column might contain non-unique values
+        because the order by column might contain non-unique values
     :param last_id: The parameter id to use as a continuation token for paging, in combination with the 'end' value,
-            because the order by column might contain non-unique values
-    :param start: The lower limit for the order by column (exclusive).
-                Only one of 'start' and 'end' should be specified at the same time.
-    :param end: The upper limit for the order by column (exclusive).
-                Only one of 'start' and 'end' should be specified at the same time.
+        because the order by column might contain non-unique values
+    :param start: The lower limit for the order by column (exclusive). Only one of 'start' and 'end' should be specified at the
+        same time.
+    :param end: The upper limit for the order by column (exclusive). Only one of 'start' and 'end' should be specified at the
+        same time.
     :param filter: Filter the list of returned parameters.
-                The following options are available:
-                name: filter by the name of the parameter
-                source: filter by the source of the parameter
-                updated: filter by the updated time of the parameter
+
+        The following options are available:
+            name: filter by the name of the parameter
+            source: filter by the source of the parameter
+            updated: filter by the updated time of the parameter
```

Comment:
In the API documentation does doesn't render as an itemization. But I am not sure whether we can do something about this.
![Screenshot from 2023-01-17 13-28-51](https://user-images.githubusercontent.com/2684622/212900689-df0feacf-dad0-4c72-8bb4-f1ada7395229.png)

Reply:
Now looks like this:
![image](https://user-images.githubusercontent.com/98876549/213183583-c5474ad2-3851-4951-bab1-a1de0eb5a54c.png)